### ISSUE TITLE
xgboost 1.6.1 not picked by the bots

### ISSUE THREAD

**Issue by ncclementi** [conda-forge/xgboost-feedstock#89]
### Comment:

I noticed that the latest version released on conda-forge was 1.5.1, I wonder when can we get a more updated and if possible the latest one released on pypi on 05/09/22.

**Comment by hcho3** [comment]
https://github.com/conda-forge/xgboost-feedstock/pull/88 is currently blocked by another issue in conda-forge.

**Comment by hcho3** [comment]
1.6.1 is now on conda-forge

**Comment by jakirkham** [comment]
Thanks Philip! 🙏

Please let us know how things go Naty 🙂

**Comment by ncclementi** [comment]
@jakirkham Thanks you for all the work here!. So far things seem to be working :)
